Jenkinsfile

```diff
@@ -13,7 +13,7 @@ pipeline {
     options {
         buildDiscarder(logRotator(numToKeepStr:'10'))
         timestamps()
-        timeout(time: 90, unit: 'MINUTES')
+        timeout(time: 180, unit: 'MINUTES')
```

Comment:
This timeout change is not needed - the underlying problem here remains that `whenBeforeInputTrue.json` is still actually the JSON equivalent of `whenBeforeInputFalse.groovy`, meaning that tests are failing and/or hanging due to the mismatch between `whenBeforeInputTrue.json` and `whenBeforeInputTrue.groovy`.